exporter/azure_monitor/azure_monitor.go

```diff
@@ -37,17 +38,87 @@ var _ trace.Exporter = (*AzureTraceExporter)(nil)
 	@param sd Span data retrieved by opencensus
 */
 func (exporter *AzureTraceExporter) ExportSpan(sd *trace.SpanData) {
-	baseObj := common.BaseObject {IKey : exporter.options.InstrumentationKey}
 	envelope := common.Envelope {
-		BaseObject: baseObj,
+		IKey : exporter.Options.InstrumentationKey,
+		Tags : common.Azure_monitor_contect,
+		Name : "Microsoft.ApplicationInsights.RemoteDependency",
+		Time : getCurrentTime(),

```

Comment:
Is this a TODO? We are supposed to take the timestamp from the span data.

Reply:
You are 100% correct. I meant to ask you about it but forgot. After looking at the python source code again, yeah it should be from the span data and that makes sense. This was an easy fix, I can still use my getCurrentTime(t time.Time) to format it properly.